code/game/machinery/computer/card.dm

```diff
@@ -321,10 +321,10 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 
 		if(length(paycheck_departments))
 			for(var/datum/bank_account/B in SSeconomy.bank_accounts)
-				var/datum/data/record/R = find_record("name", B.account_holder, GLOB.data_core.general)
+				var/datum/record/crew/R = find_record(B.account_holder, GLOB.manifest.general)
```

Comment:
```suggestion
				var/datum/record/crew/R = find_record(B.account_holder, GLOB.manifest.general)
```
this too, bad var name
but you can put this varname just be since it's obvious and readible well in this loop, but also it's an old code.